services/licence-finder/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3.7'
+
+x-licence-finder: &licence-finder
+  build:
+    context: .
+    dockerfile: Dockerfile.govuk-base
+  image: licence-finder
+  volumes:
+    - ${GOVUK_ROOT_DIR:-~/govuk}:/govuk:delegated
+    - home:/home/build
+  working_dir: /govuk/licence-finder
+
+services:
+  licence-finder-lite: &licence-finder-lite
+    <<: *licence-finder
+    privileged: true
+    depends_on:
+      - mongo
+      - elasticsearch6
+    environment:
+      MONGODB_URI: "mongodb://mongo/licence-finder_development"
+      VIRTUAL_HOST: licence-finder.dev.gov.uk
+      ELASTICSEARCH_URI: http://elasticsearch6:9200
+      HOST: 0.0.0.0
+    ports:
+      - "3000"
+
+  licence-finder-app:
+    <<: *licence-finder
+    depends_on:
+      - mongo
```

Comment:
I think we're missing an 'nginx-proxy' dependency here.

Reply:
Hmm this is part of static now but worth adding here to be explicit?

Reply:
Yup. We should add things that this app depends on directly, to avoid coupling with other services.